deployment/ccip/changeset/test_helpers.go

```diff
@@ -421,6 +416,44 @@ func CCIPSendRequest(
 	return tx, blockNum, nil
 }
 
+// retryCcipSendUntilNativeFeeIsSufficient sends a CCIP message with a native fee,
+// and retries until the fee is sufficient. This is due to the fact that the fee is not known in advance,
+// and the message will be rejected if the fee is insufficient.
+func retryCcipSendUntilNativeFeeIsSufficient(
+	e deployment.Environment,
+	r *router.Router,
+	src,
+	dest uint64,
+	msg router.ClientEVM2AnyMessage,
+) (*types.Transaction, uint64, error) {
+	const errCodeInsufficientFee = "0x07da6ee6"
+	defer func() { e.Chains[src].DeployerKey.Value = nil }()
+
+	for {
+		fee, err := r.GetFee(&bind.CallOpts{Context: context.Background()}, dest, msg)
+		if err != nil {
+			return nil, 0, errors.Wrap(deployment.MaybeDataErr(err), "failed to get fee")
+		}
+
```

Comment:
sorry I missed this. but you need to reset this value. otherwise other write calls to non payable contracts from deployment key will fail